examples/llm/resiliency/README.md

```diff
@@ -7,7 +7,7 @@ This example can be used to train a small LLaMA model using NeMo2.0 and understa
 Official documentation for these features can be found in the [NeMo user guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html).
 
 
-See [resiliency-in-pretraining-demo.ipynb](resiliency-in-pretraining-demo.ipynb) for a demo of these features.
+See [resiliency-in-pretraining-demo.ipynb](resiliency-in-pretraining-demo.ipynb) for a demo of these features. You can deploy a Launchable hands-on tutorial which is deployed on a Crusoe GPU cloud instance accelerated by NVIDIA by following​ [https://nvda.ws/42aNqnM](https://nvda.ws/42aNqnM).
```

Comment:
n00b question: does this link ever expire?

Reply:
fair question, it should not expire as long as I keep the launchable alive and I don't plan on killing it.